docker-compose-rabbitmq.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  rabbitmq:
+    #      build: ./rabbitmq/configuration
+    image: rabbitmq:3.8.3-management
+    container_name: rabbitmq_local
+    restart: always
+    hostname: "rabbitmq_local"
```

Comment:
if we use this container\_name/host\_name(rabbitmq\_local), would not change the connection string in the code?

Reply:
This will be changed as per code